service-front/module/Application/view/application/pages/how_will_the_donor_confirm.twig

```diff
@@ -1,9 +1,16 @@
+{% include 'layout/id_check_banner.twig' with details_data %}
 <div class="govuk-width-container">
     <main class="govuk-main-wrapper">
         <form method="POST">
-            <h1 class="govuk-heading-xl">How will the donor confirm their identity</h1>
+            <h1 class="govuk-heading-xl govuk-!-width-two-thirds">How will they confirm their identity</h1>
 
-            <h2 class="govuk-heading-m">ID over the phone (preferred)</h2>
+            <p class="govuk-body govuk-!-width-two-thirds">
+                Confirming their identity over the phone is the quickest and easiest option - they'll need an NI number, passport or driving licence and to answer some identity check questions during the call.
+
+                <br><br></br>Alternately, they can identify in-person at a post office - this will take longer and incurs a cost to OPG
```

Comment:
`<br>` don't need closing tags. You could also use multiple `<p>`s for this
```suggestion
                <br><br>Alternately, they can identify in-person at a post office - this will take longer and incurs a cost to OPG
```